### ISSUE TITLE
Build issue: Arch

### ISSUE THREAD

**Issue by russeree** [mgba-emu/mgba#3249]
Currently I am trying to build v0.10.3 and am getting this error. Does anyone know how I would go about resolving this? I would prefer to use the same src. that everyone else uses and not fork/patch.

```
/home/qrsnap/git/mgba/src/feature/updater-main.c: In function ‘main’:
/home/qrsnap/git/mgba/src/feature/updater-main.c:328:28: error: passing argument 2 of ‘execv’ from incompatible pointer type [-Wincompatible-pointer-types]
  328 |                 execv(bin, argv);
      |                            ^~~~
      |                            |
      |                            const char **
In file included from /home/qrsnap/git/mgba/include/mgba-util/common.h:61,
                 from /home/qrsnap/git/mgba/include/mgba/core/config.h:9,
                 from /home/qrsnap/git/mgba/src/feature/updater-main.c:6:
/usr/include/unistd.h:584:51: note: expected ‘char * const*’ but argument is of type ‘const char **’
  584 | extern int execv (const char *__path, char *const __argv[])
      |                                       ~~~~~~~~~~~~^~~~~~~~
make[2]: *** [CMakeFiles/updater-stub.dir/build.make:636: CMakeFiles/updater-stub.dir/src/feature/updater-main.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:387: CMakeFiles/updater-stub.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
```

![image](https://github.com/mgba-emu/mgba/assets/3104223/9a2f872e-89a7-4508-933b-a0eec09d4af3)

**Comment by endrift** [comment]
It's already fixed in dev and the 0.10 branch. I'm sure a "Bitcoin maximalist" is smart enough to figure out how to manage that, though given the fact that you took a photo instead of a screenshot, I'll reserve judgment on that one.

**Comment by russeree** [comment]
Yeah easy enough, I was just checking out the latest release tag currently. Thanks for the warm welcome to the project.
